### ISSUE TITLE
Nether star chicken

### ISSUE THREAD

**Issue by setycz** [setycz/ChickensMod#45]

**Comment by GenDeathrow** [comment]
lol I had the same idea about adding a nether star chicken. 

My idea was to make it so you had to craft it with 3 wither skulls and a nether star, basically kill the wither first as to not mess up progression.

**Comment by setycz** [comment]
Yeah, that sounds great ;). I would like to make it challenging as well.

**Comment by tgstyle** [comment]
Maybe make it so it drops a star only a couple times a day?

**Comment by GenDeathrow** [comment]
You could make it drop nether star nuggets. Combine to make a star.
